ci/taos/plugins-staging/pr-nnstreamer-ubuntu-apptest.sh

```diff
@@ -0,0 +1,137 @@
+    result+=$?
+    tar xvzf ./mobilenet_v1_1.0_224_quant.tgz
+    wget https://raw.githubusercontent.com/nnsuite/testcases/master/DeepLearningModels/tensorflow-lite/Mobilenet_1.0_224_quant/labels.txt
+    result+=$?
+    cd ..
+
+    if [[ ${result} -ne 0 ]]; then
+        echo -e "[DEBUG][FAILED] Oooops!!!!!! apptest is failed."
+        echo -e " The data files was not downloaded. Please check the log file to get a hint"
+        echo -e ""
+        check_result="failure"
+        global_check_result="failure"
+
+        message="Oooops. apptest is failed. Resubmit the PR after fixing correctly. Commit number is $input_commit."
+        cibot_pr_report $TOKEN "failure" "TAOS/pr-nnstreamer-apptest" "$message" "$REPOSITORY_WEB/pull/$input_pr/commits/$input_commit" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
+    
+        # comment a hint on failed PR to author.
+        message=":octocat: **cibot**: $user_id, apptest could not be completed. To find out the reasons, please go to ${CISERVER}/${PRJ_REPO_UPSTREAM}/ci/${dir_commit}/checker-pr-audit.log"
+        cibot_comment $TOKEN "$message" "$GITHUB_WEBHOOK_API/issues/$input_pr/comments"
+
+        return ${result}
+    fi
+    
+    # Test with sample apps
+    # @todo We have to modify the commented test files below because this script requires a USB camera device.
+    
+    # ./nnstreamer_example_filter
+    # python nnstreamer_example_filter.py
+    # ./nnstreamer_example_cam
+    ./nnstreamer_sink_example
+    result+=$?
+    # ./nnstreamer_sink_example_play
+
+    
+    if [[ ${result} -ne 0 ]]; then
+        echo -e "[DEBUG][FAILED] Oooops!!!!!! apptest is failed. Resubmit the PR after fixing correctly."
+        echo -e ""
+        check_result="failure"
+        global_check_result="failure"
+    else
+        echo -e "[DEBUG][PASSED] Successfully apptest is passed."
+        check_result="success"
+    fi
+    
+    echo -e "report the execution result of apptest. check_result is ${check_result}. "
+    if [[ $check_result == "success" ]]; then
+        message="Successfully apptest is passed. Commit number is '$input_commit'."
+        cibot_pr_report $TOKEN "success" "TAOS/pr-nnstreamer-apptest" "$message" "$REPOSITORY_WEB/pull/$input_pr/commits/$input_commit" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
+    else
+        message="Oooops. apptest is failed. Resubmit the PR after fixing correctly. Commit number is $input_commit."
+        cibot_pr_report $TOKEN "failure" "TAOS/pr-nnstreamer-apptest" "$message" "$REPOSITORY_WEB/pull/$input_pr/commits/$input_commit" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
+    
+        # comment a hint on failed PR to author.
+        message=":octocat: **cibot**: $user_id, apptest could not be completed. To find out the reasons, please go to ${CISERVER}/${PRJ_REPO_UPSTREAM}/ci/${dir_commit}/checker-pr-audit.log"
+        cibot_comment $TOKEN "$message" "$GITHUB_WEBHOOK_API/issues/$input_pr/comments"
+    fi
+}
+
```

Comment:
@ohsewon As we all know, the similar 2 scripts have to be maintained after this PR.  Any idea? How about we try to link this script in the existing example wiki page (Created by @jaeyun-jung ) for maintenance?